From b50f43349841643ae5fad7110fc37cb432a45208 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 31 Oct 2021 05:11:34 -0400
Subject: [PATCH 52/97] opcodes: d10v: simplify header includes

This file doesn't use anything from bfd (sysdep.h), so drop that
include.  This avoids an implicit dependency on the generated
config.h which can be problematic for build-time tools.

Also swap stdio.h for stddef.h.  This file isn't doing or using
any I/O structures, but it does need NULL.
---
 opcodes/d10v-opc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/opcodes/d10v-opc.c b/opcodes/d10v-opc.c
index c68fc6878a2..f87a0779b89 100644
--- a/opcodes/d10v-opc.c
+++ b/opcodes/d10v-opc.c
@@ -19,8 +19,7 @@
    Software Foundation, 51 Franklin Street - Fifth Floor, Boston,
    MA 02110-1301, USA.  */
 
-#include "sysdep.h"
-#include <stdio.h>
+#include <stddef.h>
 #include "opcode/d10v.h"
 
 
-- 
2.34.0

